org.jaspstats.JASP.json

```diff
@@ -0,0 +1,4046 @@
+{
+	"app-id": "org.jaspstats.JASP",
+	"runtime": "org.kde.Sdk",
+	"runtime-version": "5.12",
```

Comment:
Update runtime to 5.13

Reply:
We will only update to 5.13 after we do the same for our Windows and Mac versions and test it fully.